docs/notes/2.25.x.md

```diff
@@ -44,7 +44,7 @@ The version of Python used by Pants itself has been updated to [3.11](https://do
 - Use the package manager you used to install Pants. For example, with Homebrew: `brew update && brew upgrade pantsbuild/tap/pants`.
 - Use its built-in self-update functionality: `SCIE_BOOT=update pants`.
 
-That Pants itself happens to be partially writtin in Python has no bearing on the versions of Python that Pants can use to test and build your code.
+That Pants itself happens to be partially written in Python has no bearing on the versions of Python that Pants can use to test and build your code.
```

Comment:
"That Pants itsself" doesn't sound very grammatical. Maybe we should rephrase? I would just add "Note: The fact that ..." to the front of the sentence.